filters/filter_test.go

```diff
@@ -307,3 +307,20 @@ func TestFilters(t *testing.T) {
 		})
 	}
 }
+
+func TestString(t *testing.T) {
```

Comment:
I think this is useful to have as a test.

Can you please change the name to be more descriptive?  Maybe `TestOperatorStrings`?

Reply:
Thanks, I have changed it.

Reply:
PTAL